src/BlockNumber.js

```diff
@@ -13,15 +14,14 @@ export default function BlockNumber(props) {
     : api.derive.chain.bestNumber;
 
   useEffect(() => {
-    let unsubscribeAll;
+    let unsubscribeAll = null;
```

Comment:
What are the advantages and disadvantages of initializing to `null`? Should I learn something here?

Reply:
without null assignment, `unsubscribeAll` contains value `undefined`. 99% of the time the difference doesn't matter, like when we check using `if (!var) {...}`.

https://stackoverflow.com/questions/24811001/javascript-unassigned-var